src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -50,6 +50,9 @@ const AuctionResultsContainer: React.FC<AuctionResultsProps> = ({
     allowEmptyCreatedDates,
   } = filterContext.filters
 
+  // Count number of times user paginated.
```

Comment:
Nitpick: Is comment relevant with your latest commit to use booleans for pagination detection?

Reply:
Oh...I can see myself getting confused by that comment in future days. Corrected. Thanks!